tests/unit/test_config.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2023 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+""" test_config.py is part of csp-billing-adapter and provides units tests
```

Comment:
Nitpick: for multi-line comments we try to stick to the following formats:

Single line
```
"""This is a single line comment with no period"""
```

Multi-line
```
"""
This is the main point of a multi-line comment

And details follow after a space
"""
```

Reply:
I fixed this here and the other PR